hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/StorageImpl.java

```diff
@@ -66,6 +67,11 @@ public void clear(boolean isDuringShutdown) {
     }
 
     @Override
+    public Iterator<R> iterator(boolean failFast) {
```

Comment:
It doesn't make a lot of sense to be able to ignore the setting.

Afaik the iterator returned in this case won't suffer from concurrent modifications.

So should the documentation then not be updated to ignore the failFast=true setting?